LinuxUtils.cpp

```diff
@@ -370,8 +370,9 @@ class ConversionOutputBuffer
 		static void str(String& output, char* buffer, std::size_t length)
 		{
 			using namespace std;
```

Comment:
I think it's not needed anymore.

Reply:
It's all over the place. Since this file is ... `special`, I'm not touching it with a 20-foot pole.